packages/api/src/auth/authHeaders.ts

```diff
@@ -9,7 +9,7 @@ import { AuthenticationError } from 'apollo-server-lambda'
 
 import { verifyAuth0Token } from './verifyAuth0Token'
 
-export type SupportedAuthTypes = 'auth0' | 'netlify' | 'gotrue'
+export type SupportedAuthTypes = 'auth0' | 'netlify' | 'gotrue' | 'magicLinks'
```

Comment:
I think we should call this provider `magic`, or `magic.link`
```suggestion
export type SupportedAuthTypes = 'auth0' | 'netlify' | 'gotrue' | 'magic.link'
```